src/app/app.module.ts

```diff
@@ -27,9 +35,12 @@ import { AppRoutingModule } from './app-routing.module';
     TvShowsSearchInputComponent,
     TvShowsTvDataComponent,
     TvShowDetailComponent,
+    ProfileSettingsFormComponent,
```

Comment:
You might want to consider making a `profile.module` feature module instead of registering here. This is a good practice as the number of feature modules could grow quite large in a real world application.